docs/sources/setup-grafana/configure-grafana/feature-toggles/index.md

```diff
@@ -2,10 +2,6 @@
 aliases:
   - /docs/grafana/latest/setup-grafana/configure-grafana/feature-toggles/
 description: Learn about feature toggles, which you can enable or disable.
-labels:
```

Comment:
FYI this appeared here after the merge and conflict resolution, but it matches the current main.